### PR TITLE
Ad 53 explore alice donation contracts

### PR DESCRIPTION
**Pull request by alexAlice**

### CODE REVIEW

File: contracts/ContractProvider.sol [hunk 1]
```diff
@@ -1,4 +1,4 @@
-pragma solidity ^0.4.18;
+pragma solidity ^0.4.22;
```
Comment (jakub-wojciechowski) [inline contracts/ContractProvider.sol]: Why cannot we update to 0.4.24?

File: quarantine/integration/single-donation.js [hunk 2]
```diff
@@ -14,18 +14,18 @@ const should = require('chai')
 contract('Single donation', function(accounts) {
   var main = accounts[0];
   var donor = accounts[1];
-  var judge = accounts[3];
+  var val = accounts[3];
```
Comment (jakub-wojciechowski) [inline quarantine/integration/single-donation.js]: Does it work, you refer to this variable as "validator" later on.

File: test/helper.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  prepare: function () {
```
Comment (jakub-wojciechowski) [inline test/helper.js]: Could you please remove the prepare method and rename the module to test-setup or test-preparation, so the syntax is more concise?

**Review by jakub-wojciechowski — CHANGES_REQUESTED** [review]

File: quarantine/integration/single-donation.js (hunk 2 above)
Comment (alexAlice) [reply quarantine/integration/single-donation.js]: It doesn't work. I haven't noticed it because I have not run tests from qurantine :)

**Comment by jakub-wojciechowski** [comment]
Please update the amount of gas because after update of OpenZeppelin the base contract is more expensive and the deployment is broken: 

in app.js change to:
TokenContract = await AliceToken.new({from: aliceAccount, gas: 3000000});

**Review by jakub-wojciechowski — APPROVED** [review]

**Pull request merged**